tests/unit/httpd_test.cc

```diff
@@ -1245,6 +1245,18 @@ SEASTAR_TEST_CASE(case_insensitive_header) {
     return make_ready_future<>();
 }
 
+SEASTAR_TEST_CASE(case_insensitive_header_reply) {
+    seastar::http_response resp{};
```

Comment:
I'm killing this thing with #1657 . So please, either try to avoid using it, or let's wait for that commit to settle and then rebase

Reply:
@xemul - sounds good, you can take care of the case sensitivity issue in your PR as well then?